cmd/queue/main.go

```diff
@@ -331,10 +333,14 @@ func main() {
 	adminServer := buildAdminServer(healthState, probe, logger)
 	metricsServer := buildMetricsServer(promStatReporter)
 
+	profilingHandler := profiling.NewHandler(logger, env.EnableProfiling)
```

Comment:
Since in QP value of env.EnableProfiling never changes, what if we wrap all this in the `if` statement.
Othewise we create the server and waste resources on it, while it will never be used.

Reply:
OK. It makes sense. I did the changes.